terasoluna-gfw-functionaltest-selenium/src/test/resources/META-INF/spring/seleniumContext.xml

```diff
@@ -55,7 +55,7 @@
     </bean>
 
     <beans profile="firefox default">
-        <bean id="webDriver" class="org.openqa.selenium.firefox.FirefoxDriver"
+        <bean id="webDriver" class="org.terasoluna.gfw.functionaltest.app.FirefoxDriverFactoryBean"

```

Comment:
- [ ] This setting is  scope="prototype", but FirefoxDriverFactoryBean's  isSingleton method returns true.
Which one is correct? Probably FirefoxDriverFactoryBean's  isSingleton method returns false.